wordle-web/src/App.vue

```diff
@@ -1,46 +1,61 @@
 <template>
   <v-app>
-    <v-app-bar :elevation="3">
-      <template v-slot>
-        <v-app-bar-title>
-          <RouterLink to="/">
-            <v-icon icon="mdi-alpha-w-box" color="orange-darken-3"></v-icon>
-            Wordle
-          </RouterLink>
-        </v-app-bar-title>
-        <v-spacer></v-spacer>
+    <v-parallax src="https://cdn.vuetifyjs.com/images/parallax/material.jpg">
+      <v-app-bar :elevation="3" color="yellow-lighten-1">
+        <template v-slot>
+          <v-app-bar-title>
+            <RouterLink to="/wordoftheday">
+              <v-icon icon="mdi-alpha-w-box" color="cyan"></v-icon>
+              Daily Word
+            </RouterLink>
+          </v-app-bar-title>
+          <v-spacer></v-spacer>
 
-        <v-btn icon="mdi-brightness-7" @click="switchTheme"></v-btn>
+          <v-btn icon="mdi-brightness-7" color="cyan" @click="switchTheme"></v-btn>
 
-        <ActiveUser></ActiveUser>
+          <ActiveUser></ActiveUser>
 
-        <v-menu>
-          <template v-slot:activator="{ props }">
-            <v-btn icon="mdi-hamburger" v-bind="props"></v-btn>
-          </template>
+          <v-menu>
+            <template v-slot:activator="{ props }">
+              <v-btn icon="mdi-hamburger" color="cyan" v-bind="props"></v-btn>
+            </template>
 
-          <v-list>
-            <v-list-item>
-              <v-list-item-title>
-                <RouterLink to="/">Play Again</RouterLink>
-              </v-list-item-title>
-            </v-list-item>
-            <v-list-item>
-              <v-list-item-title>
-                <RouterLink to="/leaderboard">Leaderboard</RouterLink>
-              </v-list-item-title>
-            </v-list-item>
-            <v-list-item>
-              <v-list-item-title><RouterLink to="/about">About</RouterLink></v-list-item-title>
-            </v-list-item>
-          </v-list>
-        </v-menu>
-      </template>
-    </v-app-bar>
+            <v-list>
+              <v-list-item>
+                <v-list-item-title>
+                  <RouterLink to="/wordoftheday">Word of the day</RouterLink>
+                </v-list-item-title>
+              </v-list-item>
+              <v-list-item>
+                <v-list-item-title>
+                  <RouterLink to="/">Practice mode</RouterLink>
+                </v-list-item-title>
+              </v-list-item>
+              <v-list-item>
+                <v-list-item-title>
+                  <RouterLink to="/leaderboard">Leaderboard</RouterLink>
+                </v-list-item-title>
+              </v-list-item>
+              <v-list-item>
+                <RouterLink to="/instructions">Instructions</RouterLink>
+              </v-list-item>
+              <v-list-item>
+                <v-list-item-title
+                  ><RouterLink to="/stats">Statistics</RouterLink></v-list-item-title
```

Comment:
```suggestion
                  > <RouterLink to="/stats"> Statistics </RouterLink> </v-list-item-title
```
If you add spaces between these, your linter will format the code a bit nicer!